src/styles/index.scss

```diff
@@ -22,7 +22,7 @@ h2 {
 }
 
 #top-fold {
-	background: url(../images/backgrounds/space-background.jpg) no-repeat;
+	background: url(../images/backgrounds/home-background.jpg) no-repeat;
```

Comment:
I am not a huge fan of the image personally, but I can live with it if Hannah gives it the okay. I don't see a diff in here for removing `space-background.jpg`, and I'm okay leaving it in the repo for now, but generally speaking, it's good to remove assets that you aren't using to help keep the repo size and asset library small.

Reply:
per Hannah's background img request of photo of campus